P/PAPI/build_tarballs.jl

```diff
@@ -42,9 +56,43 @@ products = [
     LibraryProduct("libpapi", :libpapi)
 ]
 
-# Dependencies that must be installed before this package can be built
-dependencies = Dependency[
+cuda_versions_to_build = Any[
+    v"10.2",
+    v"11.0",
+    "none"
+]
+
+# XXX: support only specifying major/minor version (JuliaPackaging/BinaryBuilder.jl#/1212)
+cuda_versions = Dict(
+    v"10.2" => v"10.2.89",
+    v"11.0" => v"11.0.3",
+)
+
+cuda_platforms = [
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
 ]
 
-# Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6")
+for cuda_version in cuda_versions_to_build, platform in platforms
+    tag = cuda_version == "none" ? "none" : CUDA.platform(cuda_version)
+    cuda_version != "none" && !(platform in cuda_platforms) && continue
+    augmented_platform = Platform(arch(platform), os(platform);
+                                  libc=platform.libc,
+                                  cuda=tag)
+    should_build_platform(triplet(augmented_platform)) || continue
+
+    if cuda_version == "none"
+        dependencies = []
+    else
+        if arch(platform)
+        dependencies = [
+            BuildDependency(PackageSpec(name="CUDA_full_jll",
+                                        version=cuda_full_versions[cuda_version])),
+            RuntimeDependency(PackageSpec(name="CUDA_Runtime_jll")),
+        ]
```

Comment:
I think this should be
```suggestion
            dependencies = [
                BuildDependency(PackageSpec(name="CUDA_full_jll",
                                            version=cuda_full_versions[cuda_version])),
                RuntimeDependency(PackageSpec(name="CUDA_Runtime_jll")),
            ]
        end
```
?